### ISSUE TITLE
Gestures for Mac version

### ISSUE THREAD

**Issue by immyneedscake** [Layendan/NineAnimator-Tauri#34]
Can you add the capability for a Two-finger swipe left-right to map to forward-back button? It will be a nice feature to be consistent with other mac apps and browsers.

**Comment by Layendan** [comment]
Yeah I love that feature as well, afaik that is not possible with the technologies that I am using. I'll try to figure out something, but I don't believe that it's possible to implement.

**Comment by immyneedscake** [comment]
one alternate could be to have a shortcut mapped to it like (⌘+[ to go back and ⌘+] to go forward) so that way if users want, they can map those shortcuts to gestures using BetterTouchTool

**Comment by Layendan** [comment]
> one alternate could be to have a shortcut mapped to it like (⌘+[ to go back and ⌘+] to go forward) so that way if users want, they can map those shortcuts to gestures using BetterTouchTool

Currently I have it set to ⌘ (macos) / alt (windows) + ← to navigate backwards and ⌘/alt + → to navigate forwards although I could also add [ and ] to the shortcuts.

**Comment by immyneedscake** [comment]
Awesome, mapped them to BTT and its a great workaround!
